Calendar_project/calendar.js

```diff
@@ -0,0 +1,160 @@
+const calendarContainer = document.querySelector(".calendar-container");
+const todayContainer = calendarContainer.querySelector(".current-day");
+const todaysDate = todayContainer.querySelectorAll("p")[0]
+const todaysDateNum = todayContainer.querySelectorAll("p")[1]
+const currentMonth = calendarContainer.querySelector(".current-month");
+const calendarBody = document.querySelector(".calendar-body");
+
+//현재 날짜
+let today = new Date();
+let thisYear = today.getFullYear();
+let nowMonth = today.getMonth();
+let nowDay = today.getDay(); //요일
+let nowDate = today.getDate(); //날짜
+
+const monthEng = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "July", "Aug", "Sep", "Oct", "Nov", "Dec"];
+const weekdayEng = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
+//const weekdayKr = ["일요일", "월요일", "화요일", "수요일", "목요일", "금요일", "토요일"];
+
+todaysDate.innerHTML = weekdayEng[nowDay];
+todaysDateNum.innerHTML = nowDate;
+currentMonth.innerText = monthEng[nowMonth] + " " + thisYear;
+
+//현재 년의 월 일 1일의 일과 요일 표시
+let firstDay = new Date(today.getFullYear(), today.getMonth(), 1);
+let firstDayName = firstDay.getDay();
+
+let monthDays = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+//현재월 마지막 일
+let lastDay = monthDays[today.getMonth()];
+//달력의 필요한 행의 갯수
+let row = Math.ceil(((lastDay + firstDayName) / 7));
+
+function printCalendar() {
+    let monthCount = 1;
+    let dayDefault = 1;
+    let year = today.getFullYear();
+    let month = today.getMonth();
+    const date = new Date();
+
+    if ((today.getFullYear() % 4 == 0 && today.getFullYear() % 100 != 0) || (today.getFullYear() % 400 == 0)) {
+        monthDays[1] = 29;
+    }
+    for (i = 0; i < row; i++) {
```

Comment:
for문의 initialization 부분은 let으로 선언해주는 것이 좋습니다. 이후 같은 코멘트는 생략하겠습니다!
```suggestion
    for (let i = 0; i < row; i++) {
```